Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeInteractionRegionLayers.mm

```diff
@@ -98,6 +108,10 @@ void updateLayersForInteractionRegions(CALayer *layer, const RemoteLayerTreeTran
 
     bool applyBackgroundColorForDebugging = [[NSUserDefaults standardUserDefaults] boolForKey:@"WKInteractionRegionDebugFill"];
 
+    auto* node = RemoteLayerTreeNode::forCALayer(layer);
+    if (!node || webViewIsLinkPreview(node->uiView()))
```

Comment:
This is a very late time to have this check; at this point we've already wasted lots of time constructing the regions. I think it would be better to just have SPI that disables the InteractionRegionsEnabled WebPreference.

Reply:
Good idea, I’ll do that instead